### DISCUSSION TITLE
This repository is temporarily unavailable.

### DISCUSSION THREAD

**Discussion by Jen-ogo** [community/community#19094] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello. I have been having the same problem in GitHub for today, and I last edited my code today. Whenever I open my repository, it says it is temporarily unavailable and it is “moving to a newer, better home.” I have tried refreshing the page multiple times, and the same message appears and my code doesn’t appear. I have attached a screenshot of what appears when I open my repository. It would be great if someone could help me to delete this repo. Thanks!
<img width="1405" alt="Screenshot 2022-06-21 at 22 47 22" src="https://user-images.githubusercontent.com/78457125/174894871-7fe8d1d7-c63b-48ca-bd43-d7f1ad455a65.png">

**Comment by melvincwng** [comment, marked as answer]
@Jen-ogo, please kindly raise a ticket to [GitHub Support](https://support.github.com/) for the support engineers to investigate the issue.

They usually respond pretty quickly (within a couple of working days).

Hope this clarifies.

If you find this reply useful, please kindly 'Mark as answer' so that others know this issue is already addressed.

Thanks and have a good day ahead 😄

**Reply by Jen-ogo** [reply]
Thanks a lot.

**Reply by choi-junhee** [reply]
I'm suffering the same issue.

**Reply by choi-junhee** [reply]
How can I raise the ticket? Please let me know. Or is there already the ticket?

**Comment by Grey-Wind** [comment]
I'm suffering the same issue.

**Comment by tvcsantos** [comment]
I'm having the same problem
